pkg/parser/digester.go

```diff
@@ -95,9 +95,9 @@ func Normalize(sql string) (result string) {
 // which removes general property of a statement but keeps specific property.
 //
 // for example: NormalizeForBinding('select 1 from b where a = 1') => 'select ? from b where a = ?'
-func NormalizeForBinding(sql string) (result string) {
+func NormalizeForBinding(sql string, isPlanReplayer bool) (result string) {
```

Comment:
how about changing the var name as `forPlanReplayerReload`, which will be more clear here

Reply:
the deep call chain name is the same